applications/nrf_desktop/configuration/nrf52kbd_nrf52832/prj_release_fast_pair.conf

```diff
@@ -132,4 +132,5 @@ CONFIG_BT_ATT_TX_COUNT=6
 CONFIG_BT_CONN_TX_MAX=6
 
 CONFIG_BT_FAST_PAIR=y
+CONFIG_BT_FAST_PAIR_USE_CASE_INPUT_DEVICE=y
```

Comment:
Maybe we could comment that there is currently no keyboard use-case, so that generic input device use-case is used?

Reply:
sure, I will add it together with documentation